pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -189,14 +190,16 @@ let
       ++ extraMakeFlags;
 
       installFlags = [
-        "INSTALLKERNEL=${installkernel}"
         "INSTALL_PATH=$(out)"
       ] ++ (optional isModular "INSTALL_MOD_PATH=$(out)")
       ++ optional installsFirmware "INSTALL_FW_PATH=$(out)/lib/firmware"
       ++ optionals buildDTBs ["dtbs_install" "INSTALL_DTBS_PATH=$(out)/dtbs"];
 
       preInstall = ''
         installFlagsArray+=("-j$NIX_BUILD_CORES")
+
+        # the install scripts expect to find installkernel in ~/bin/installkernel
+        export HOME=${installkernel}
```

Comment:
If I understand correctly, this replaces `INSTALLKERNEL=...`. Is there a reason this is better? It feels like setting HOME because something else somewhere expects the path to be just right is very brittle. On the other hand `INSTALLKERNEL=${installkernel}` seems pretty explicit.

Reply:
The previous solution only worked by accident. The upstream install scripts look for `~/bin/${INSTALLKERNEL}` and `/sbin/${INSTALL KERNEL}`, which after the mass-replacement turned into `${HOME}${INSTALLKERNEL}` and `${INSTALLKERNEL}` respectively, and the latter happened to work with what we were passing in. We could patch the path directly into the install scripts, but it's declared in different locations in different kernel releases, so this is easier.

Reply:
Could we make this a bit clearer by adding the reasoning in the comment, also linking to e.g. the current LTS's `installkernel` usage as an example of why this is needed?

Reply:
I am very much hoping we will not have this in 1+ year, but sure, I'll add a comment.

Reply:
Removed. Better now than in 1+ year.

Reply:
Unremoved. The obvious thing doesn't work.